ui/eureka/eureka.html

```diff
@@ -0,0 +1,43 @@
+<html>
+<head>
+  <meta charset="utf-8" />
+  <!-- OverlayPlugin widget resize handle -->
+  <link rel="stylesheet" href="../../resources/defaults.css">
+  <link rel="stylesheet" href="../../resources/resize_handle.css">
+  <script src="../../resources/resize_handle.js"></script>
+  
+  <style>
+    #eurekaId { 
+      width: calc(10/6*100%);
+      height: calc(10/6*100%);
+      border: 0px; 
+      zoom: 0.6;
+      -moz-transform: scale(0.6);
+      -moz-transform-origin: 0 0;
+      -o-transform: scale(0.6);
+      -o-transform-origin: 0 0;
+      -webkit-transform: scale(0.6);
+      -webkit-transform-origin: 0 0;
+      opacity: 0.5;
+    }
+  </style>
+  <script>
+    document.addEventListener("onZoneChangedEvent", function (e) {
+	    if (e.detail.zoneName == 'Eureka Anemos') 
+	      document.getElementById("eurekaId").style.display = "block";
+	    else
+	  	document.getElementById("eurekaId").style.display = "none";
+    });
+    document.addEventListener("onLogEvent", function(e) {
+      for (var i = 0; i < e.detail.logs.length; i++) {
+        var eurekaId = e.detail.logs[i].match('ffxiv-eureka.com/(.{6})');
```

Comment:
Oh neat, I was thinking about doing something like this myself, but I found the eureka tracker to be way too big for my screen.  I was going to try to pull out that information into a different format somehow.  Do you run ffxiv at a really high screen resolution?

At the very least, can you pull this eureka stuff out into a separate pull request? I generally like to have pull requests (and commits) be logically related.  If you're doing a bunch of translations, those can all go together.  But translations and eureka are separate in my mind.

Reply:
this was just a test.
i zoomed the overlay to 60% and it supports the resize handle.
but a more overlay friendly display of the information would be nice.